json_encoder_test.go

```diff
@@ -66,6 +88,7 @@ func TestJSONAddString(t *testing.T) {
 
 func TestJSONAddBool(t *testing.T) {
 	withJSONEncoder(func(enc *jsonEncoder) {
+		enc.AddString("foo", "bar")
```

Comment:
should we just delete the `"foo": "bar"` rather than calling `AddString` in these tests? Not sure what the purpose of the `AddString` is